fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -3349,6 +3350,67 @@ ACTOR Future<Void> processTransactionStateRequestPart(TransactionStateResolveCon
 
 } // anonymous namespace
 
+//
+// Metrics related to the commit proxy are logged on a five second interval in
+// the `ProxyMetrics` trace.  However, it can be hard to determine workload
+// burstiness when looking at such a large time range.  This function adds much
+// more frequent logging for certain metrics to provide fine-grained insight
+// into workload patterns. The metrics logged by this function break down into
+// two categories:
+//
+//   * existing counters reported by `ProxyMetrics`
+//   * new counters that are only reported by this function
+//
+// Neither is implemented optimally, but the data collected should be helpful
+// in identifying workload patterns on the server.
+//
+// Metrics reporting by this function can be disabled by setting the
+// `BURSTINESS_METRICS_LOG_INTERVAL` knob to a negative value.
```

Comment:
Can we rely on an explicit flag to enable/disable this, instead of a negative value for the interval?